config/dev.webpack.config.js

```diff
@@ -4,7 +4,19 @@ const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPl
 const config = require('@redhat-cloud-services/frontend-components-config');
 const { config: webpackConfig, plugins } = config({
     rootFolder: resolve(__dirname, '../'),
-    debug: true
+    debug: true,
+    useProxy: process.env.USEPROXY ? true : false,
+    localChrome: process.env.INSIGHTS_CHROME,
+    customProxy: process.env.USEPROXY ? [
+        {
+            context: (path) => path.includes('/api/'),
+            target: 'https://cloud.redhat.com',
+            secure: true,
+            changeOrigin: true,
+            autoRewrite: true,
+            ws: true
+        }
+    ] : []
```

Comment:
This only referes to prod api, I think you could have something like
```JS
customProxy: process.env.API_ENDOINT ? [
        {
            context: (path) => path.includes('/api/'),
            target: process.env.API_ENDOINT,
            secure: true,
            changeOrigin: true,
            autoRewrite: true,
            ws: true
        }
    ] : []
```

And when running the app `API_ENDOINT=https://cloud.redhat.com npm start` or for CI api `API_ENDOINT=https://ci.cloud.redhat.com npm start`

Reply:
good point! tbh, i dont want to type that out every time, but encapsulating this mod with yer second comment would allow for maximum flexibility, thanks for lookin @karelhala 🤗